orb-ui/cone/src/lcd.rs

```diff
@@ -96,20 +94,18 @@ async fn handle_lcd_update(
         .map_err(|e| eyre::eyre!("Error flushing display: {:?}", e))?;
 
     loop {
-        tokio::select! {
-            _ = shutdown_receiver.changed() => {
-                debug!("LCD task shutting down");
-                return Ok(())
+        select! {
+            _ = &mut kill_rx => {
+                // gracefully terminate
+                tracing::trace!("lcd task killed");
```

Comment:
How I think about log levels:

info: informational messages that do get sent to datadog
debug: informational messages that don't get sent to datadog by default
trace: high frequency spam

IMO this falls into the "debug" category.